features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -126,6 +126,8 @@
 
   "oauth.dcrm.application_role_permission_required_to_view": true,
 
+  "oauth.allow_additional_params_from_error_url": false,
```

Comment:
we can remove the default value as this is properly handled in the code. adding the default value will always add it to the identity.xml file.

Reply:
done